digits/model/tasks/torch_train.py

```diff
@@ -295,7 +297,7 @@ def process_output(self, line):
         if match:
             index = float(match.group(1))
             l = match.group(2)
-            assert not('inf' in l or 'nan' in l), 'Network reported %s for training loss. Try decreasing your learning rate.'  % l
+            assert not('inf' in l or 'nan' in l), 'Network reported %s for training loss. Try changing your learning rate.'  % l
```

Comment:
You should remove this diff.
`digits.model.images.generic.test_views.TestTorchCreation.test_diverging_network` is failing because the test expects `Try decreasing your learning rate` but here DIGITS throws `Try changing your learning rate`.